apps/docs/src/pages/components/dropdown-menu.mdx

```diff
@@ -0,0 +1,102 @@
+import {
+  dropDownMenuClasses,
+  HvDropDownMenu,
+} from "@hitachivantara/uikit-react-core";
+import { Open, Preview, Save } from "@hitachivantara/uikit-react-icons";
+
+import Playground from "@docs/components/code/Playground";
+import { Description } from "@docs/components/page/Description";
+import { Page } from "@docs/components/page/Page";
+import { getComponentData } from "@docs/utils/component";
+
+export const getStaticProps = async ({ params }) => {
+  const meta = await getComponentData(
+    "DropDownMenu",
+    "core",
+    dropDownMenuClasses,
+  );
+  return { props: { ssg: { meta } } };
+};
+
+<Description />
+
+<Page>
+
+<Playground
+  Component={HvDropDownMenu}
+  componentName="HvDropDownMenu"
+  controls={{
+    placement: {
+      defaultValue: "right",
+    },
+    variant: {
+      defaultValue: "secondaryGhost",
+    },
+    size: {
+      defaultValue: "md",
+    },
+    disabled: {
+      defaultValue: false,
+    },
+    keepOpened: {
+      defaultValue: false,
+    },
+  }}
+  componentProps={{
+    dataList: [{ label: "Save" }, { label: "Export" }, { label: "Preview" }],
+  }}
+/>
+
+### Customized items
+
+The items in the `dataList` prop can be can be customized with icons,
```

Comment:
missing something? or was it meant to be a `.`?
maybe we coud mention `disabled` and "custom content" labels?